dbt/models/marts/sections/dim_section_instructors.sql

```diff
@@ -0,0 +1,42 @@
+-- model: dim_section_instructors
+-- scope: invited co-teachers
```

Comment:
This is not limiting to 'invited co-teachers'.
```suggestion
-- scope: all teachers who created sections since 10/17/2023 when the co-teacher feature was added to the platform. 
```